### ISSUE TITLE
example code

### ISSUE THREAD

**Issue by ofZach** [borg/ofxGist#1]
just an FYI that I'm battling compiling the example, since it uses an unspecified version of ofxOpenALSoundPlayer (it turns out that it's yours....) and maybe some changes to the current master on ofxHistoryPlot (there's a function I am missing), and there are two version of the KISS library (once in ofxOpenALSoundPlayer and one in ofxGist). 

also, there's stuff in the example like "loadSong("assets/sounds/"+parts[0],true);" which doesn't really correlated 

I'm going to try stripping down the example and maybe not using all these things, just want to give you a heads up that the example is a little hard to get up and running !   

am def excited to give this a spin...

**Comment by borg** [comment]
Right, I can clean that up a little. The ofxOpenALSoundPlayer is only there because I couldn't find a way to get the buffer data out of the normal ofSoundPlayer...is there (short of sending it out and back in through ofSoundStream)? 

As for duplicate kisses (doesn't sound like a problem IRL ;-) yeah, but they both use it. I thought I added a note about removing one instance, but maybe it got lost. Can add  a note to readme.

Oh has @armadillo been modifying ofxHistoryPlot again? Which function was missing. ofxGist doesn't depend on any other library, but the example would leave little so see without these. I can add a ofSoundStream only one though.

Also, chroma and chord analysis parts aren't really included yet, but TBD.

I am learning about how to use MFCC for analysis of timbre, and will be able to offer some insights of that as some stage.

**Comment by synesthete** [comment]
@borg The ofxHistoryPlot error seems to be related to the `plots[v]->getVariableName()` on line 230 of ofApp.cpp.

**Comment by borg** [comment]
@synesthete right, thanks. I  have updated the files to point to the right versions. Cheers.
[ofxHistoryPlot](https://github.com/local-projects/ofxHistoryPlot)
